includes/actions-filters.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Various theme and plugin actions and filters
+ */
+
+/**
+ * Remove the "slim" class from the array of body classes.
+ *
+ * @param array $classes
+ *
+ * @return array
+ *@since 1.0.0
+ *
+ */
+function sc_remove_slim_body_class( $classes = array() ) {
+
+	$found = array_search( 'slim', $classes );
+
+	if ( is_page() && ( false !== $found ) ) {
+		unset( $classes[ $found ] );
+	}
+
+	if ( is_front_page() && ( 1 === get_theme_mod( 'sc_show_promotional_hero', 0 ) ) ) {
+		$classes[] = 'has-promotional-hero';
+	}
+
+	if ( true === sc_discount_promo_is_active() ) {
+		$classes[] = 'promo-is-active';
+	}
+
+	return $classes;
+}
+add_filter( 'body_class', 'sc_remove_slim_body_class', 11 );
+
+/**
+ * Add promotional hero area to front page
+ */
+function sc_promotional_hero() {
+	$promo_is_active = 1 === get_theme_mod( 'sc_show_promotional_hero', 0 ) ? true : false;

```

Comment:
I don't think this conditional will ever validate.

You're trying to compare `1` strictly to `true`.

Reply:
Not sure I follow. Sanitization and the default value of the customizer option will give me either `1` for checked, or `0` for anything else. This ternary operator checks for `1` against `1`. If it passes, `true` is set as the value of `$promo_is_active`. Otherwise, it's set to `false`. Am I misunderstand what you're referring to, or perhaps how this all works?

Reply:
@SDavisMedia sorry, I read the order of operations wrong. Verified this does work as intended.